src/auth/src/credentials.rs

```diff
@@ -551,6 +551,7 @@ fn adc_well_known_path() -> Option<String> {
 // Skipping mutation testing for this module. As it exclusively provides
 // hardcoded credential stubs for testing purposes.
 #[cfg_attr(test, mutants::skip)]
+#[doc(hidden)]
```

Comment:
Auth probably wants to offer some form of insecure credentials. They are useful for customers when testing a real client against an emulator, for example.

Thoughts?

Reply:
A cred that just returns back an empty headermap?

Reply:
> A cred that just returns back an empty headermap?

Yeah. This is not a necessity, though.

Applications that need it could trivially implement `CredentialsProvider`. We would just be saving them some time by writing that code for them.